src/lib/util/is-email.ts

```diff
@@ -1,6 +1,6 @@
 // Email address matcher.
 // eslint-disable-next-line no-useless-escape
-const matcher = /.+\@.+\..+/;
+const matcher = /.[^\n\r@\u2028\u2029]*@.+\..+/;
```

Comment:
Id like som tests on isEmail, seems scary to me to change.

Reply:
Fair point :+1: I thought we had some, but I'll double check

Reply:
I've changed the implementation to use a rather flexible allow list with an extensive list of allowed characters: `[A-Z0-9.!#$%&'*+-/=?^_{|}~]` rather than accepting all and excluding some